engine/source/materials/mat/mat124/sigeps124.F

```diff
@@ -96,6 +96,7 @@ SUBROUTINE SIGEPS124(
      .   KAPDC2,FSC,ALPHAC,SIGTXX,SIGTYY,SIGTZZ,SIGTXY,SIGTYZ,SIGTZX,
      .   SIGCXX,SIGCYY,SIGCZZ,SIGCXY,SIGCYZ,SIGCZX,ELXX,ELYY,ELZZ,ELXY,
      .   ELYZ,ELZX,SIGOXX,SIGOYY,SIGOZZ,SIGOXY,SIGOYZ,SIGOZX,FT1
+      my_real, PARAMETER :: MAXDAM = 0.999D0
```

Comment:
Could you replace MAXDAM by an already existing constant? You can use ZEP999 or ONE - EM03.

Reply:
Hi, 

Thanks for the review. Changes have been implemented. 

Valentin.